force-app/main/service/DonationHistoryService.cls

```diff
@@ -100,7 +100,26 @@ public with sharing class DonationHistoryService {
         } else {
             throw new UTIL_Permissions.InsufficientPermissionException(Label.commonInsufficientPermissions);
         }
-        
+    }
+
+    /** 
+    * @description method to return Donation History list.
+    * @param paymentListForOpportunity List<npe01__OppPayment__c> for current opportunity
+    * @return String with payment method for current opportunity
```

Comment:
Technically it returns the Payment Method for the first Paid Payment, unless none are paid, then it returns the first Payment's Payment Method, right? Let's update this!